src/sentry/static/sentry/app/components/actions/ignore.jsx

```diff
@@ -119,13 +119,13 @@ export default class IgnoreActions extends React.Component {
           windowChoices={this.getIgnoreWindows()}
         />
         <div className="btn-group">
+          <GuideAnchor target="ignore_delete_discard" type="text" />
```

Comment:
Any visual differences when moving this?

Reply:
Not that I could see. The green ripple was still overlapping the ignore button.